qa/L0_perf_analyzer/perf_analyzer_profile_export_schema.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Is this a duplicate of the schema in the PA repo? We shouldn't have two copies of this file.

Reply:
The schema will be stored here, since it's used for testing. The duplicate from the client repo was removed from @debermudez's PR in this commit:

https://github.com/triton-inference-server/client/pull/375/commits/36b0c75818134cbd4e011f75783d6c771e1cd3b6